rma/models/stock_warehouse.py

```diff
@@ -56,8 +56,8 @@ def _get_rma_location_values(self):
             "location_id": self.env.ref("rma.stock_location_rma").id,
         }
 
-    def _get_sequence_values(self):
-        values = super()._get_sequence_values()
+    def _get_sequence_values(self, name=False, code=False):
+        values = super()._get_sequence_values(name, code)
```

Comment:
Always pass the keyword arguments as keyword:

```suggestion
        values = super()._get_sequence_values(name=name, code=code)
```